### PR TITLE
Update directory structure and package names

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
$subject related to https://github.com/ballerina-platform/ballerina-standard-library/issues/1623

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/tests/connectionless_client_tests.bal [hunk 1]
```diff
@@ -113,10 +113,10 @@ isolated function prepareDatagram(string msg, string remoteHost = "localhost", i
 }
 
 public function startUdpServer() returns error? = @java:Method 
-{'class: "org/ballerinalang/stdlib/udp/testutils/MockServerUtils"} external;
+{'class: "io/ballerina/stdlib/udp/testutils/MockServerUtils"} external;
```
Comment (ThisaruGuruge) [inline ballerina/tests/connectionless_client_tests.bal:116]:
```suggestion
{'class: "io.ballerina.stdlib.udp.testutils.MockServerUtils"} external;
```

File: build.gradle [hunk 2]
```diff
@@ -23,7 +23,7 @@ plugins {
 }
 
 allprojects {
-    group = 'org.ballerinalang'
+    group = 'io.ballerina.stdlib'
```
Comment (ThisaruGuruge) [inline build.gradle:26]:
```suggestion
    group = project.group
```

**Comment by ThisaruGuruge** [comment]
Ideally, this shouldn't decrease the code coverage. Please check the codecov.yml file whether there's any mistake.

File: ballerina/tests/connectionless_client_tests.bal (hunk 1 above)
Comment (Maninda) [reply ballerina/tests/connectionless_client_tests.bal]: Changed all such instances.

File: build.gradle (hunk 2 above)
Comment (Maninda) [reply build.gradle]: Changed.

**Comment by Maninda** [comment]
> Ideally, this shouldn't decrease the code coverage. Please check the codecov.yml file whether there's any mistake.

This seems to be due to runtime related race condition which seems not be due to the an issue of the code coverage related configuration.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**